apps/cowswap-frontend/src/modules/fortune/containers/FortuneWidget/index.tsx

```diff
@@ -288,18 +327,18 @@ const StyledCloseIcon = styled(X)`
   }
 `
 
-export function FortuneWidget() {
+interface FortuneWidgetProps {
+  menuTitle?: string
+}
+
+export function FortuneWidget({ menuTitle }: FortuneWidgetProps) {
   const { openFortune } = useAtomValue(fortuneStateAtom)
   const lastCheckedFortune = useAtomValue(lastCheckedFortuneAtom)
   const updateOpenFortune = useSetAtom(updateOpenFortuneAtom)
   const [isFortunesFeatureDisabled, setIsFortunesFeatureDisabled] = useAtom(isFortunesFeatureDisabledAtom)
   const openRandomFortune = useOpenRandomFortune()
   const [isNewFortuneOpen, setIsNewFortuneOpen] = useState(false)
   const [isFortunedShared, setIsFortunedShared] = useState(false)
-
-  const [today, setToday] = useState(new Date())
-  useInterval(() => setToday(new Date()), 2_000)
```

Comment:
@shoom3301 @alfetopito would you know why this userInterval was in place? It causes the Fortune banner to re-render every 2 seconds when opened on mobile/responsive.

Reply:
From the code it looks like it's done to keep the date updated.
I wasn't involved so I can't say why that's important.